src/haddock/modules/scoring/mdscoring/__init__.py

```diff
@@ -84,15 +99,6 @@ def _run(self):
 
         output_fname = "mdscoring.tsv"
         self.log(f"Saving output to {output_fname}")
-        with open(output_fname, "w") as fh:
-            fh.write(f"structure\toriginal_name\tscore{linesep}")
-            for pdb in self.output_models:
-                # temporary solution to get the original name
-                #  here one .pdb = one .psf and the .psf is not changed by
-                #  the module so use its name as the original one
-                original_name = pdb.topology.file_name.replace('.psf', '.pdb')
-                fh.write(
-                    f"{pdb.file_name}\t{original_name}\t{pdb.score}{linesep}"
-                    )
-
+        self.output(self.output_models, output_fname)
```

Comment:
If `self.output` is a method of the module and `self.output_models` is defined, better to have `output` as a normal method and internally use the `self.output_models`.
If you want to keep it in a separate function, better have a single function inside the `scoring/` folder that is used by both modules. I see `self.output` is the same for both `emscoring  and `mdscoring`. I can handle it if you want.

Reply:
agreed and implemented! I went for the first option, as probably there will not be other functions shared by both the scoring modules. If those will be implemented, the second option is definitely better :)